webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/TagsEditForm.scala

```diff
@@ -56,8 +57,8 @@ class TagsEditForm(tags: Tags, objectId: String) extends Loggable {
       }
       var scope = angular.element($$("#${controllerId}")).scope();
       scope.$$apply(function(){
-        scope.init(  ${jsTags}, "${filterId}" ,  ${isEditForm}, ${isRule}, "${objectId}");
+        scope.init(  ${jsTags}, "${filterId}" ,  ${isEditForm}, ${isRule}, "${StringEscapeUtils.escapeEcmaScript(objectId)}");
       });
-    """)))
+    """))) // JsRaw ok, escaped
```

Comment:
Found this along the way : by testing random JS characters in directive id, I got a console error in the Ajax response at this line :see_no_evil: